website/docs/components/side-nav/partials/guidelines/overview.md

```diff
@@ -1 +1,7 @@
-The SideNav provides users with access to the main pages of the product and should be used as the primary navigation in the application.
\ No newline at end of file
+The Side Nav provides users with access to contextual navigation within areas of the application and generally includes subpages and nested pages within projects and organizations.
+
+!!! Info
+
+Some elements that were previously contained within the Side Nav, including the home link, help dropdown, user dropdown, and context switcher, have been migrated to the [App Header](/components/app-header). To avoid introducing a breaking change, we’ve created a Side Nav - v2.0 component that should be used along with App Header. In Ember, the elements removed were yielded in the component, incurring no breaking change.
```

Comment:
A few points
- _"we’ve created a Side Nav - v2.0 component"_  -  only in Figma, it should be clarified
- _"the elements removed were yielded in the component, incurring no breaking change"_ - as developer is not clear what this changes for me: have you removed the yielded components (in which case it is a breaking change) or not (in which case, we should not even mention Ember here)

Reply:
This actually needs to be updated based on the change in naming convention.